### ISSUE TITLE
Can the encoder of seq2seq model be  replaced with other encoding models not mentioned? 

### ISSUE THREAD

**Issue by tensixchuan** [microsoft/CodeBERT#98]
May the encoder part of the model, in addition to the given options (such as codebert-base, roberta-base), can be replaced with other encoding models?

**Comment by guoday** [comment]
yes. You just need to change `"microsoft/codebert-base"` in the command. The options include `"microsoft/graphcodebert-base"`, `"roberta-base"` and `"roberta-large"`
